File: Starters/C#/4.3 - Test Test Patterns/TestingPatterns/models/ProductFactory.cs
using System;
using System.Collections.Generic;
using System.Text;

public class ProductFactory
{
    public static Product randomProduct()
    {
        Random rand = new Random();

        double price = 1.0 + rand.NextDouble() * 99;
        return new Product("Product" + rand.Next(1,100), price);
    }
}

File: Starters/Java/4.3 - Testing patterns/src/models/ProductFactory.java
package models;

import java.util.Random;

public class ProductFactory {
    private static final Random rand = new Random();

    public static Product randomProduct() {
        double price = 1.0 + rand.nextDouble() * 99;
        return new Product("Product" + rand.nextInt(1000), price);
    }
}
